funflow/TestFunflow.hs

```diff
@@ -32,22 +35,33 @@ flow2caught = retry 100 0 flow2
 flow3 :: SimpleFlow [Int] [Int]
 flow3 = mapA (arr (+1))
 
+runFailingFlow :: Path Abs Dir -> IO ()
+runFailingFlow storeDir = withSimpleLocalRunner storeDir $ \runner -> do
+   r <- runner cachedFailStep ()
+   print r
+
+testFailingFlow :: IO ()
+testFailingFlow =
+  withSystemTempDir "test_output" $ \storeDir ->
+    runFailingFlow storeDir >> runFailingFlow storeDir
+
 main :: IO ()
-main =
+main = do
+  testFailingFlow
   withSystemTempDir "test_output" $ \storeDir ->
-  CS.withStore storeDir $ \store -> do
-    memHook <- createMemoryCoordinator
-    res <- runSimpleFlow MemoryCoordinator memHook store flow2 ()
-    print res
-    res' <- runSimpleFlow MemoryCoordinator memHook store flow2caught ()
-    print res'
-    putStrLn $ showFlow myFlow
-    putStrLn $ showFlow flow2
-    res1 <- runSimpleFlow MemoryCoordinator memHook store flow3 [1..10]
-    print res1
---  main = redisTest
-    externalTest
-    storeTest
+    CS.withStore storeDir $ \store -> do
+      memHook <- createMemoryCoordinator
+      res <- runSimpleFlow MemoryCoordinator memHook store flow2 ()
+      print res
+      res' <- runSimpleFlow MemoryCoordinator memHook store flow2caught ()
+      print res'
+      putStrLn $ showFlow myFlow
+      putStrLn $ showFlow flow2
+      res1 <- runSimpleFlow MemoryCoordinator memHook store flow3 [1..10]
+      print res1
+  --  main = redisTest
```

Comment:
Should that comment be removed?

Reply:
Possibly, but I didn't edit it as part of this PR